gradle/shipkit.gradle

```diff
@@ -0,0 +1,31 @@
+shipkit {
+   gitHub.repository = "linkedin/dynamometer"
+
+   //TODO generate and use your own read-only GitHub personal access token
+   gitHub.readOnlyAuthToken = "76826c9ec886612f504d12fd4268b16721c4f85d"
+
+   //TODO generate GitHub write token, and ensure your Travis CI has this env variable exported
+   gitHub.writeAuthToken = System.getenv("GH_WRITE_TOKEN")
+}
+
+allprojects {
+   plugins.withId("com.jfrog.bintray") {
```

Comment:
I see in the example that it should be `org.shipkit.bintray`, where does `com.jfrog` come from?

Reply:
That's what `./gradlew initShipkit` added, not sure why 🤔